src/cc65/funcdesc.h

```diff
@@ -72,6 +72,7 @@ struct FuncDesc {
     struct FuncDesc*    FuncDef;        /* Descriptor used in definition     */
     struct SymEntry*    WrappedCall;    /* Pointer to the WrappedCall        */
     unsigned char       WrappedCallData;/* The WrappedCall's user data       */
+    int                 WrappedCallUseBank;/* Flag: does WrappedCall use .bank() or literal value */
```

Comment:
Perhaps there is a better way to add flags. Please elaborate :)

Reply:
A "boolean" variable makes sense here.  The code to handle it is simpler than the code would be for any other way to add that flag.

Reply:
I'm surprised: There's already `unsigned Flags` in line 66. All that would be necessary would be to define an additional `FD_...` macro. However, I suggest to get rid of a flag/boolean value alltogether - see my other comment.